auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -396,8 +396,7 @@ export default class AccountInfo extends Mixins(
   ]
 
   private async mounted () {
-    const accountSettings = this.getAccountFromSession()
-    await this.syncOrganization(accountSettings.id)
+    await this.syncOrganization(parseInt(this.$route.params['orgId']))
```

Comment:
Are you sure orgId is always passed as a param?

Reply:
Yes